### PR TITLE
update CI, format

### PR DESCRIPTION
**Pull request by kevinlang**

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -1,24 +1,48 @@
...
-jobs:
-  test-elixir:
-    runs-on: ubuntu-latest
-    container: elixir:${{ matrix.elixir_version }}-slim
 
+on:
+  pull_request:
+  push:
+    branches:
+      - master
```
Comment (cw789) [inline .github/workflows/ci.yml:7]: Default branch is `main`.

**Review by cw789 — COMMENTED** [review]
Probably overseen.

File: .github/workflows/ci.yml (hunk 1 above)
Comment (kevinlang) [reply .github/workflows/ci.yml:7]: nice catch :)

**Comment by warmwaffles** [comment]
Looks good @kevinlang

**Pull request merged**